pkg/reconciler/kubernetes/tektonpipeline/controller.go

```diff
@@ -57,6 +57,7 @@ func NewExtendedController(generator common.ExtensionGenerator) injection.Contro
 			logger.Fatalw("Error creating client from injected config", zap.Error(err))
 		}
 		mflogger := zapr.NewLogger(logger.Named("manifestival").Desugar())
+		// FIXME(vdemeester) why ignoring error ?
 		manifest, _ := mf.ManifestFrom(mf.Slice{}, mf.UseClient(mfclient), mf.UseLogger(mflogger))
```

Comment:
no reason. it should atleast be logged.
I shall take care of it in a different patch in all NewExtendedController Functions. :+1: